max.lua

```diff
@@ -212,7 +212,7 @@ function module.display_clients(s,direction)
     awful.client.focus.byidx(direction == "right" and 1 or -1)
     capi.client.focus:raise()
   end
-  local clients = awful.client.tiled(s)
```

Comment:
I am not sure about this change. It might have some side effects. Can you split this into 2 PRs? One for the `local s` and `current_idx` and one for that? I don't have a lot of time right now, so I would like to merge the change that's obviously correct and eventually take time to test this part.